rules/windows/process_creation/proc_creation_win_sysinternals_sysmon_config_update.yml

```diff
@@ -0,0 +1,28 @@
+title: Sysmon Configuration Update
+id: 87911521-7098-470b-a459-9a57fc80bdfd
+status: test
+description: Detects updates to Sysmon's configuration. Attacker might update or replace Sysmon configuration with a bare bone one to avoid monitoring without shutting down with the service permanently
+references:
+    - https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon
+author: Nasreddine Bencherchali (Nextron Systems)
+date: 2023/03/09
+tags:
+    - attack.defense_evasion
+    - attack.t1562.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_pe:
+        - Image|endswith:
+            - \Sysmon64.exe
+            - \Sysmon.exe
+        - Description: 'System activity monitor'
+    selection_cli:
+        CommandLine|contains:
+            - '-c'
+            - '/c'
+    condition: all of selection_*
+falsepositives:
+    - Legitimate administrators might use this command to update Sysmon configuration.
+level: medium
```

Comment:
I would prefer it as a hunting rule. Normally you would whitelist the configuration hash of your accepted configs before deploying them. --> Environment specific rule needed for higher levels.

For checking the config state I would use ID 16 over process creation and give a template to show how to add to the list of accepted config hashes. This rule could then be medium for general purposes but high or even critical in actively maintained environments.

Reply:
We could even add the Nextron and SwiftOnSecurity Sysmon config hashes as default values and add them on config updates

Reply:
There is actually a rule using EID 16 for config changes and it's already high. https://github.com/SigmaHQ/sigma/blob/master/rules/windows/sysmon/sysmon_config_modification_status.yml

This is more like a variant. But I like the idea of the config hash.

I will update the old rule to include instructions on how to add the hash and add the hash of both ours and swift's configs.

Reply:
Ok. Seems like no one had an issue with that so far. Do people not update their Sysmon config? I still don't *like* medium for this rule but I am willing to accept it.